.circleci/config.yml

```diff
@@ -12,6 +12,18 @@ install-cf7: &install-cf7
       rm cf7.deb
       cf7 api https://api.fr.cloud.gov
 
+run-common-tasks: &run-common-tasks
+# these common tasks run on all spaces: dev, staging and prod
+  steps:
+    - run:
+      # Refresh the materialized view table for trends
+        name: Refresh trends view
+        command: cf run-task crt-portal-django  -c "python crt_portal/manage.py refresh_trends" --name refresh-trends
+      # Clear expired sessions in session table
+    - run:
+        name: Clear expired sessoins
+        command: cf run-task crt-portal-django  -c "python crt_portal/manage.py clearsessions" --name clear-sessions
+
```

Comment:
This isn't quite right and would break as written, let's revert this piece and break this limit the scope of this pull request to running the periodic tasks on staging and production. We'll tackle refactoring in a separate issue.

Reply:
@Jkrzy Changes made according to the scope of this pull request to running the periodic tasks on staging and production. CI/CD steps format corrected. Please review.